packages/suite/src/support/suite/useTor.tsx

```diff
@@ -1,28 +1,34 @@
 import { useEffect } from 'react';
 import { desktopApi } from '@trezor/suite-desktop-api';
 import { useActions, useSelector } from '@suite-hooks';
-import { isTorDomain, torFetch } from '@suite-utils/tor';
+import { getIsTorEnabled, isTorDomain as getIsTorDomain, torFetch } from '@suite-utils/tor';
```

Comment:
nit: I would rather choose different name for variable?

Reply:
I wanted to be consistent with the `getIsTorEnabled`. Any suggestions? :)

Reply:
My suggestion still is that not all the bool-returning functions must be named `getIs***`.

Reply:
0905e85e7